mlflow/tracking/fluent.py

```diff
@@ -815,6 +815,26 @@ def get_experiment_by_name(name: str) -> Optional[Experiment]:
     return MlflowClient().get_experiment_by_name(name)
 
 
+def list_experiments(
+    view_type: int = ViewType.ACTIVE_ONLY, max_results: Optional[int] = None,
+) -> List[Experiment]:
+    """
+        :param view_type: Qualify requested type of experiments.
+        :param max_results: If passed, specifies the maximum number of experiments desired. If not
+                            passed, all experiments will be returned.
+        :param page_token: Token specifying the next page of results. It should be obtained from
+                            a ``list_experiments`` call.
```

Comment:
This isn't part of the API definition. Can we remove it?

Reply:
Thanks for the catch! Will remove it.